py_zipkin/_encoding_helpers.py

```diff
@@ -195,3 +208,208 @@ def copy_endpoint_with_new_service_name(endpoint, new_service_name):
         ipv6=endpoint.ipv6,
         port=endpoint.port,
     )
+
+
+def get_encoder(encoding):
+    """Creates encoder object for the given encoding.
+
+    :param encoding: desired output encoding protocol.
+    :type encoding: Encoding
+    :return: corresponding IEncoder object
+    :rtype: IEncoder
+    """
+    if encoding == Encoding.V1_THRIFT:
+        return _V1ThriftEncoder()
+    if encoding == Encoding.V1_JSON:
```

Comment:
nit: `elif`?

Reply:
There's a return before this, so it's not really useful. I actually got a low-priority warning when using the elif.